client/src/components/ShoppingLists.js

```diff
@@ -46,7 +46,7 @@ const ShoppingLists = ()=>{
                 </Typography>
                 <br/>
                 <Grid container spacing={5}>
-                {shListsContext.lists.map((list) => (
+                {shListsContext.lists.slice().reverse().map((list) => (
```

Comment:
We can use this for now. Actually we have created date as a parameter in List schema. We can sort descending through Mongoose and pass data to FE alternatively.

Reply:
Great! I will try.